src/PastePage.js

```diff
@@ -0,0 +1,76 @@
+/* global fetch */
+import React, { Component } from 'react'
+import PasteBox from './pasteBox'
+import PasteInlineForm from './pasteInlineForm'
+/*
+import {
+  Form,
+  FormGroup,
+  FormControl,
+  Button
+} from 'react-bootstrap'
+*/
+
+class PastePage extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      codeText: '',
+      codeTitle: '',
+      codeAuthor: '',
+      langSelect: '',
+      skillSelect: ''
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.sendToFB = this.sendToFB.bind(this)
+  }
+
+  handleChange (e, formName) {
```

Comment:
check it out, here's something I learned recently. If you declare your function like this:
```
handleChange = (e, formName) => {
  this.setState({[formName]: e.target.value})
}
```
then you don't have to `.bind()` it!